content/_changelogs/12.0.0.md

```diff
@@ -1,20 +1,139 @@
 ## 12.0.0
 
-_Released MM/DD/YYYY_
+_Released 12/06/2022_
+
+**Summary:** ...Notes below...
 
 **Breaking Changes:**
 
-- `experimentalSessionAndOrigin` flag has been removed and all functionality is
-  on by default now.
-- The new concept of `testIsolation` has been introduced and it defaults to
-  `on`.
-- Cypress now throws an error if commands are invoked from inside a `.should()`
-  callback. This previously resulted in unusual and undefined behavior; it is
-  now explicitly an error.
-- `Cookies.defaults` and `Cookies.preserveOnce` have been removed. Please update
-  to use [`cy.session()`](/api/commands/session) to preserve session details
+- Cypress dropped support for Node 12. Use Node 14, Node 16 or Node 18+.
+  Addressed in [#24885](https://github.com/cypress-io/cypress/pull/24885).
```

Comment:
The other entries are using `Addresses`
```suggestion
  Addresses [#24885](https://github.com/cypress-io/cypress/pull/24885).
```

Reply:
I think the difference is this ones a PR. The issues are "Addresses" and the PRs are "Addressed in"